docs/logs/faq.md

```diff
@@ -606,28 +606,23 @@ later.
 
 ### What are Logs App Alerts?
 
-Logs App alerts are based on saved searches that trigger alert
-notifications when these saved searches meet a certain threshold
+Logs App alerts are based on full text search queries and filters. They trigger alert
+notifications when these query and filters meet a certain threshold
```

Comment:
notifications when these query and filters meet a certain threshold
=>
notifications when the query and filters meet a certain threshold